### ISSUE TITLE
Sound popping in some games

### ISSUE THREAD

**Issue by bmaupin** [libretro/TempGBA-libretro#4]
I saw this core was [based off TempGBA4PSP-26731013.](https://github.com/libretro/TempGBA-libretro/commit/a336dc05302ba6a6c8112e43cde1e44798c476dd), so I did some [quick comparisons](https://gist.github.com/bmaupin/dac967cd0c527cab028a3bb5c2c11762) and I noticed this core exhibits issues in some games that TempGBA4PSP-26731013 doesn't seem to have.

In particular, the sound pops very badly in Metroid: Zero Mission during normal gameplay with this core, but in TempGBA4PSP-26731013 the sound is fine.

Similarly, the sound pops very badly in Mother 3 during battle scenes with this core but not with TempGBA4PSP-26731013.

Is this something that even could be fixed? I've read libretro can be slower than native emulators, but since this core is only for PSP I would expect it to run as well as a native emulator.

Thanks!

**Comment by twinaphex** [comment]
Libretro cores.are.in general not slower.than native.emulators. That is propaganda pushed by some trying to paint a certain narrative.

**Comment by bmaupin** [comment]
@twinaphex That's good to know!

I did some more testing and interestingly enough my testing supports your statement; it seems like TempGBA4PSP and TEMPGBA4PSP-mod are actually using frame skipping to work around the audio popping issues.

Is frame skipping something that would be accepted as a PR if it was implemented for this core? It would probably be reasonable for it to be off by default since most games run fine. I know it's not an ideal solution but at least it would make these games playable until someone is able to solve this in a proper way (if that's even possible given that the PSP is such an old and underpowered device).

Thanks!
